src/h5web/vis-packs/core/shared/Html.tsx

```diff
@@ -92,15 +119,16 @@ const Html = forwardRef<HTMLDivElement, HtmlProps>((props, ref) => {
       </div>,
       el
     );
-  }, [children, className, el, style, ref, divProps]);
+  }, [children, className, divProps, el, getInnerDivTransform, style]);
```

Comment:
The `getInnerDivTransform` is now an explicit dependency, which is much cleaner. This is why it has to be a `useCallback`.